app/views/public/profile.html.erb

```diff
@@ -13,7 +13,7 @@
         <%= render partial: "public/mysavedpost"%>
```

Comment:
use snake case for file names when there are more than one word

Reply:
Resolved by creating new Branch #113